### PR TITLE
Feature #252 - Сhange add order endpoint for several orders

### LINKED ISSUE #252: SP: 8 Change add order endpoint to make it work for several services at a time
Part of the User Story #39

### PR DESCRIPTION
**Pull request by Misha86**

### CODE REVIEW

File: beauty/api/serializers/order_serializers.py [hunk 1]
```diff
@@ -30,33 +32,43 @@ class Meta:
...
+        request = self.context.get("request")
+        specialist = attrs.get("specialist")
+        start_time = attrs.get("start_time")
+        service = attrs.get("service")
+
+        errors = {}
+        if timezone.now() > start_time:
+            logger.info("The start time should be more as now.")
+
+            errors.update({"start_time": "The start time should be more as now."})
```
Comment (pazuzu-ua) [inline beauty/api/serializers/order_serializers.py]: "should be later then now"

File: beauty/api/serializers/order_serializers.py [hunk 2]
```diff
@@ -30,33 +32,43 @@ class Meta:
...
-        specialist = validated_data.get("specialist")
-        customer = validated_data.get("customer")
+        request = self.context.get("request")
+        specialist = attrs.get("specialist")
+        start_time = attrs.get("start_time")
+        service = attrs.get("service")
+
+        errors = {}
+        if timezone.now() > start_time:
+            logger.info("The start time should be more as now.")
```
Comment (pazuzu-ua) [inline beauty/api/serializers/order_serializers.py]: "should be later then now"

**Review by pazuzu-ua — APPROVED** [review]

File: beauty/api/serializers/order_serializers.py (hunk 1 above)
Comment (Misha86) [reply beauty/api/serializers/order_serializers.py]: Thanks.

File: beauty/api/serializers/order_serializers.py (hunk 2 above)
Comment (Misha86) [reply beauty/api/serializers/order_serializers.py]: done

**Review by OleksiiDatsiuk — APPROVED** [review]

File: beauty/api/tests/test_order_serializers.py [hunk 3]
```diff
@@ -74,9 +79,26 @@ def test_invalid_serializer(self):
...
+                        "service": self.service.id}
+
+        serializer = self.Serializer(data=invalid_data, context={"request": self.request})
+        serializer.is_valid()
+
+        self.assertFalse(serializer.is_valid())
+        self.assertEqual(serializer.validated_data, {})
+        self.assertEqual(serializer.data, invalid_data)
+        self.assertEqual(serializer.errors, {
+            "start_time": [ErrorDetail(string="The start time should be more as now.",
```
Comment (datsiuk-ira) [inline beauty/api/tests/test_order_serializers.py]: same as Dmytro mention

**Review by datsiuk-ira — CHANGES_REQUESTED** [review]

File: beauty/api/tests/test_order_serializers.py (hunk 3 above)
Comment (Misha86) [reply beauty/api/tests/test_order_serializers.py]: done

**Review by datsiuk-ira — APPROVED** [review]

**Pull request merged**